src/patternfly/layouts/Grid/docs/code.md

```diff
@@ -1,14 +1,14 @@
 ## Overview
 
-A Grid layout framework including responsive columns, row spans and gutters 
+A Grid layout framework including responsive columns, row spans and gutters. **Breakpoint is optional. Breakpoints: -on-xs, -on-sm, -on-md, -on-lg, -on-xl**
 
 ## Usage
 
 | Class | Applied To | Outcome |
 | -- | -- | -- |
 | `.pf-l-grid` | `<div>` | Initializes the grid layout. |
 | `.pf-l-grid__item` | `<div>` | Explicitly sets a child of the grid. This class isn't necessary, but it is included to keep inline with BEM convention, and to provide an entity that will later be used for applying modifiers. |
-| `.pf-m-gutters` | `.pf-l-grid` | Adds space between children |
-| `.pf-m-{1-12}-col` | `.pf-l-grid__item` | Defines grid item width.  Although not required, they are strongly suggested. If not used, grid item will default to 12 col. |
-| `.pf-m-{1-12}-col-on-{breakpoint}` | `.pf-l-grid__item` | Defines grid-item width based on breakpoint.  Breakpoints: xs, sm, md, lg, xl |
-| `.pf-m-{2-x}-row`, `.pf-m-{1-11}-col` | `.pf-l-grid__item` | Defines grid item row span.  For row spans to function correctly, the value of of the current row plus the grid items to span must be equal to or less than 12. Example: .pf-m-8-col.pf-m-2-row + .pf-m-4-col + .pf-m-4-col. There is no limit to number of spanned rows. |
+| `.pf-m-gutters` | `.pf-l-grid` | Adds grid-gap. |
```

Comment:
is it strange to have the modifier say gutter when the property is called gap? Maybe it doesn't matter.

Reply:
I agree, probably should change to `.pf-m-grid-gap` and add `.pf-m-column-gap` and `.pf-m-row-gap`

Reply:
Even though we know the property is a grid gap the thing that generates is a gutter. This is also shared with the different layouts that don't use grid. For example flex and margins on split. They all share the same modifier: gutter